core-bundle/contao/templates/modules/mod_navigation_accessible.html.twig

```diff
@@ -68,12 +69,23 @@
         .mergeWith(script_attributes|default)
     %}
 
+    {% add "accessible_navigation_js" to body %}
+        <script{{ script_attributes }}>
+            const accessibleNavBreakpoint = parseInt(window.getComputedStyle(document.body).getPropertyValue('--nav-breakpoint'), 10) || 1024;
```

Comment:
```suggestion
            const accessibleNavBreakpoint = parseInt(window.getComputedStyle(document.body).getPropertyValue('--nav-breakpoint'), 10) ?? 1024;
```

This way somebody can set it to zero, if they only want the desktop version. wdyt?

Reply:
If we only ever want the desktop version, we would not need the ResizeObservers at all + not checking for the window.width. I would rather enhance the JS and disable observing completely + not use those checks but that would be another PR imho.